model/repo_ref.go

```diff
@@ -275,6 +278,14 @@ func (r *RepoRef) UpdateAdminRoles(toAdd, toRemove []string) error {
 	return nil
 }
 
+func (r *RepoRef) removeFromAdmins(user string) {
```

Comment:
You shouldn't need this function; RepoRef inherits the methods from ProjectRef since it just wraps it (this might not be true since it's private but you could just make it public). I'd also say "removeFromAdminsList" to make it clear that this is only local.

Reply:
done